src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public Colori getRandomColor() {
+
+        int randomIndex = new Random().nextInt(Colori.values().length);
```

Comment:
The `Random` object is created each time the `getRandomColor` method is called. This is not efficient because creating a new `Random` object each time is unnecessary. Instead, you can create a single `Random` object as a class field and reuse it. This is a performance issue.